ansible/roles/liquid-core/templates/local.py

```diff
@@ -19,3 +19,4 @@
 
 HOOVER_APP_URL = 'http://hoover.{{ liquid_domain }}'
 HYPOTHESIS_APP_URL = 'http://hypothesis.{{ liquid_domain }}'
+DOKUWIKI_APP_URL = 'http://dokuwiki.{{ liquid_domain }}'
```

Comment:
Eh, I just pushed [the patch](https://github.com/liquidinvestigations/core/commit/bf5df3d), it was straightforward. The other `*_APP_URL`s are not documented either.